mne/_freesurfer.py

```diff
@@ -537,3 +539,62 @@ def _get_lut(fname=None):
     lut = {d[0]: np.array(lut[d[0]], dtype=d[1]) for d in dtype}
     assert len(lut['name']) > 0
     return lut
+
+
+def get_head_surface(surf, subject, subjects_dir, bem=None):
```

Comment:
You need a `@fill_doc` decorator here for the `%(subject)s` to fill in. But really you want `*, verbose=None` at the end and `@verbose` because some of the subfunctions call `logger.info`.